### ISSUE TITLE
Missing 02-blocks.ipynb

### ISSUE THREAD

**Issue by Bachstelze** [udapi/udapi-python#109]
The file for the tutorial [02-blocks.ipynb](https://nbviewer.jupyter.org/github/udapi/udapi-python/blob/master/tutorial/01-blocks.ipynb) is missing.

**Comment by martinpopel** [comment]
Yes, unfortunately I have not even started writing that part of the tutorial. Maybe in autumn.
That said, you can write me an email what do you need Udapi for and I will try to reply with some hints.

I've now removed the links 02-blocks.ipynb and marked it as "not finished yet". I will keep this issue open as another reminder to finish the tutorial.
